app/src/main/resources/database/scripts/initial-schema.sql

```diff
@@ -6,4 +6,12 @@ CREATE TABLE recipe (
 	target_vg smallint NOT NULL,
 	base_nicotine smallint NOT NULL,
 	quantity smallint
+);
+
+CREATE TABLE recipe_flavourings (
```

Comment:
The table name is great, but let's use as our standard the table names in the singular ```recipe_flavouring```. But there's nothing wrong with using the plural, many teams adopt the plural standard, I just think that some words will read a bit weird when they are put in the plural and we would have to worry about the English plural rules like for example status -> statuses.